sw/device/tests/sim_dv/i2c_host_tx_rx_test.c

```diff
@@ -59,16 +59,21 @@ static volatile const uint8_t kI2cIdx = 0;
 static volatile bool fmt_irq_seen = false;
 static volatile bool rx_irq_seen = false;
 static volatile bool done_irq_seen = false;
+/**
+ * these variables store values based on kI2cIdx
+ */
+static volatile uint32_t i2c_irq_fmt_watermark_id;
+static volatile uint32_t i2c_base_addr;
+static volatile top_earlgrey_plic_irq_id_t plic_irqs[9];
```

Comment:
do these need to be volatile? they are only set by the SW in the test, no?

Reply:
yes, they're only set once in the SW. Removed volatile. Thanks